src/main/kotlin/domain/Cell.kt

```diff
@@ -5,11 +5,31 @@ enum class Cell(
 ) {
     CLOSED("C"),
     MINE("*"),
+    ZERO("0"),
+    ONE("1"),
+    TWO("2"),
+    THREE("3"),
+    FOUR("4"),
+    FIVE("5"),
+    SIX("6"),
+    SEVEN("7"),
+    EIGHT("8"),
     ;
 
+    fun isMine(): Boolean = this == MINE
+
+    fun isClosed(): Boolean = this == CLOSED
+
     companion object {
+        private val neighborMineCountRange: IntRange = 0..8
```

Comment:
주변 지뢰 갯수에 대해서 잘 구현해주셨네요. 👍 
다만, enum을 수정하면 자동으로 변경될 수 있도록 변경해보면 어떨까요?
```suggestion
        private val neighborMineCountRange: IntRange = ZERO.symbol.toInt()..EIGHT.symbol.toInt()
```

Reply:
네 말씀하신 의견 좋은것같아요 반영했습니다 😀